### PR TITLE
Cassandra: add Flow using unlogged batches

### PR DESCRIPTION
**Pull request by beikern**
* Add flow that uses unlogged batches to improve throughput when stream elements share the same 
partition key
* Implements tests for both Scala and Java.
* Documentation updated describing the new flow.

### CODE REVIEW

**Comment by beikern** [comment]
Fixed a typo in ```CassandraBatchSettings``` sorry for that :)

File: cassandra/src/main/scala/akka/stream/alpakka/cassandra/CassandraBatchSettings.scala [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+object CassandraBatchSettings {
+  val Defaults = CassandraBatchSettings(50, 500.millis)
+}
+final case class CassandraBatchSettings(maxGroupSize: Int, maxGroupWait: FiniteDuration) {
+  require(
+    maxGroupSize > 0,
+    s"Invalid value for maxGroupSize: $maxGroupSize. It should be > 0."
+  )
+}
```
Comment (ennru) [inline cassandra/src/main/scala/akka/stream/alpakka/cassandra/CassandraBatchSettings.scala]: Please add Java-friendly `withMaxGroupSize` and `withMaxGroupWait(Long, TimeUnit)` methods.

**Comment by beikern** [comment]
@ennru I agree with you about the ```Sink``` version. I've removed the implementation, test and docs using the sink.

The metohds ```withMaxGroupSize``` and ```withMaxGroupWait(Long, TimeUnit)``` are implemented too.

Tell me if you think there's some issue that needs to be addressed.

Greetings.

File: cassandra/src/main/scala/akka/stream/alpakka/cassandra/CassandraBatchSettings.scala (hunk 1 above)
Comment (beikern) [reply cassandra/src/main/scala/akka/stream/alpakka/cassandra/CassandraBatchSettings.scala]: Done.

**Comment by beikern** [comment]
I've removed unused imports in both scala and java cassandra sinks and rebased master. Hope it's ready to be merged!

**Comment by ennru** [comment]
Thank you for this contribution. Keep going!

**Pull request merged**